README.md

```diff
@@ -1,16 +1,18 @@
 ## Flyt - (aka VSM/Value Stream Mapping)
 
-![Flyt canvas (13) - Process feedback on 'Flyt'  - Wednesday, September 15, 2021](https://user-images.githubusercontent.com/3164065/133470524-d6934d90-82ce-4870-aea3-450fd1f4c48f.png)
+![image](https://github.com/equinor/flyt/assets/115637685/b2cbc217-1b39-4668-b891-afed82edcd01)
```

Comment:
Shall we use the screenshot from Flyt 2.0 instead? And the screenshot doesn't include all cards in process 13?

Reply:
I've replaced an old screenshot with a new one from Flyt 2.0, but probably that was not from the process 13. It doesn't matter wich process we're using for this purpose, I believe